src/pseudopeople/configuration/validator.py

```diff
@@ -317,7 +317,7 @@ def validate_noise_level_proportions(
     # If there is no data for a queried dataset, we want the user's to hit the correct error that there
     # is no data available so we do not throw an error here.
     if dataset_noise_proportions.empty:
```

Comment:
```suggestion
    if not dataset_noise_proportions.empty:
```

removes the need to have the first branch at all, if I'm reading correctly